lib/modules/metadata.mjs

```diff
@@ -73,7 +75,18 @@ export async function validateCategory (doc, { mode = MODES.NORMAL } = {}) {
 
   switch (doc.type) {
     case 'txt': {
-      // TODO: Text type validation
+      const docCategory = doc.data.header.category || doc.data.header.intendedStatus
+      const docName = doc.data.slug
+
+      if (!docName.startsWith('draft-') && !docCategory) {
+        result.push(new ValidationWarning('MISSING_DOC_CATEGORY', 'The document category attribute is missing on the RFC element.', {
+          ref: 'https://authors.ietf.org/en/rfcxml-vocabulary#category'
+        }))
+      } else if (docCategory && !rfcStatusHierarchy.find(item => item.name.toLowerCase() === docCategory.toLowerCase())) {
+        result.push(new ValidationWarning('INVALID_DOC_CATEGORY', 'The document category has an invalid value. Allowed values are std, bcp, info, exp and historic.', {
```

Comment:
Instead of "std, bcp, info, exp and historic" please write full status names, because short one is used in xml and full in txt